scripts/linters/pylint_extensions.py

```diff
@@ -540,11 +595,60 @@ def check_functiondef_params(self, node, node_doc):
             node_doc, node.args, node,
             accept_no_param_doc=node_allow_no_param)
 
-    def check_arg_section_indentation(self, node):
-        """Checks whether the function argument definitions ("Args": section)
-        are indented properly. Parameters should be indented by 4 relative to
-        the 'Arg: ' line and any wrap-around descriptions should be indented
-        by 8. The rest of the docstring section checks are in DocstringChecker.
+    def check_docstring_section_style(self, node):
+        """Checks whether the docstring has valid style.
```

Comment:
What is a valid style? It looks too general. Can we please make the check name more precise and also change the docstring to say something like `it fetches a node.... which is passed to check_docstring_structure which checks  whether the docstring has a space at the beginning and a period at the end`.

Reply:
Done.